Documentation/network/concepts/ipam/multi-pool.rst

```diff
@@ -0,0 +1,153 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    http://docs.cilium.io
+
+.. _ipam_crd_multi_pool:
+
+Multi-Pool (Beta)
+#################
```

Comment:
Given that it's beta, let's add the standard disclaimer?

```suggestion
Multi-Pool (Beta)
#################

.. include:: ../../../beta.rst
```

(Same for `ipam-multi-pool.rst`)